Directory.Packages.props

```
@@ -16,6 +16,7 @@
     <PackageVersion Include="coverlet.collector" Version="6.0.2" />
     <PackageVersion Include="Basic.Reference.Assemblies" Version="1.7.9" />
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
+    <PackageVersion Include="MudBlazor" Version="7.8.0" />
```

Please use the latest version, 8.9.0.

```suggestion
    <PackageVersion Include="MudBlazor" Version="8.9.0" />
```